### DISCUSSION TITLE
2018 data not pulling?

### DISCUSSION THREAD

**Discussion by Jfreundlich01** [cwendt94/espn-api#173] [ESPN API] [answered]
Hey wondering if others are able to access 2018 data? Everything else is working great! Thanks! Hope some of y'all won your leagues this year haha!

**Comment by cwendt94** [comment, marked as answer]
Hmm that seems odd! I am able to access 2018 for my league. What error are you getting?

**Reply by Jfreundlich01** [reply]
hm weird. Not sure if it would be helpful to post the full error code.

It is seems to be an error with _fetch_league(), espn_request.get_league(), get_league, league_get, checkingRequestStatus. At the end it says "Exception: Access Denied". 

This would lead me to assume it has to do with the actual ESPN League, but I am able to access data from every other year with the same line of code, so not sure. 

Thanks for the help!

**Reply by cwendt94** [reply]
One possibility is that in 2018 your league was private so to access you will need to pass in your credentials.

**Reply by Jfreundlich01** [reply]
Yup, that worked. Sorry should have tried that before raising the issue. Strange that one of our years was private. Appreciate the support!!

**Reply by cwendt94** [reply]
No problem, glad it worked!
